Sources/Logto/Core/LogtoCore+Fetch.swift

```diff
@@ -60,4 +71,31 @@ extension LogtoCore {
             completion(nil, error)
         }
     }
+
+    /// Fetch token by `refresh_token`.
+    /// Note the func will NOT validate any token in the response.
+    static func fetchToken(
+        useSession session: NetworkSession = URLSession.shared,
+        byRefreshToken refreshToken: String,
+        tokenEndpoint: String,
+        clientId: String,
+        resource: String? = nil,
+        scope: ValueOrArray<String>? = nil,
+        completion: @escaping HttpCompletion<CodeTokenResponse>
+    ) {
+        let body: [String: Any] = [
+            "grant_type": TokenGrantType.refreshToken.rawValue,
+            "refresh_token": refreshToken,
+            "client_id": clientId,
+            "resource": resource as Any,
+            "scope": scope?.inArray.joined(separator: " ") as Any,
+        ]
+
+        do {
+            let data = try JSONSerialization.data(withJSONObject: body)
+            Utilities.httpPost(useSession: session, endpoint: tokenEndpoint, body: data, completion: completion)
+        } catch {
+            completion(nil, error)
```

Comment:
Sorry my bad did a deep dive and run some tests on the `fetchToken` both with authorization_code and refresh_token. With & without resource & scopes provided. I'll update the SDK conversions and docs tomorrow. Let's hold this PR for now. 

1. fetchToken by authorizaiton_code takes optional resource input params as well.  And no scopes
2. fetchToken by refresh token takes both optional resource input and scopes input. If the scope params missing openid scope. no idtoken will be issued align with the refresh token.